CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace CanHazFunny
+{
+	public class Jester : IJokeService, IJokeOutput
+	{
+		// redundant _jokeService field, but else leads to infinite regress
+		private IJokeService _jokeService;
```

Comment:
Field should be underscore PascalCase
```suggestion
		private IJokeService _JokeService;
```